src/components/organisms/TimingDeleteModal/TimingDeleteModal.tsx

```diff
@@ -75,25 +77,29 @@ export const TimingDeleteModal: React.FC<TimingDeleteModalProps> = ({
 
   return (
     <Modal show={show} onHide={onHide}>
-      <div className="form-container">
+      <div className="form">
```

Comment:
For consistency with other components.

One way would be:

```suggestion
      <div className="TimingDeleteModal TimingDeleteModal__form">
```

although that leaves the top level component not the same class as this component, so

another way would be:

```suggestion
      <div className="TimingDeleteModal">
        <Modal show={show} onHide={onHide}>
          <div className="TimingDeleteModal__form">
```

Reply:
done 917110d